.github/workflows/integrationTest.yml

```diff
@@ -61,11 +165,10 @@ jobs:
           -var="ami=${{ matrix.arrays.ami }}"
           -var="ca_cert_path=${{ matrix.arrays.caCertPath }}"
           -var="arc=${{ matrix.arrays.arc }}"
+          -var="binary_name=${{ matrix.arrays.binaryName }}"
 
       - name: Terraform destroy
-        if: always()
+        if: ${{ always() && steps.ec2-linux-integration-test.outputs.cache-hit != 'true' }}
```

Comment:
Do we need the `always()` here if we are including the cache hit check?

Reply:
Yes since if we do not include it and the test fails this will not run. Always forces a run even if a step before fails. The cache hit make sure this does not run on cache hit.